mysql-test/mysql-test-run.pl

```diff
@@ -317,7 +317,8 @@ END
 our $opt_valgrind= 0;
 my $opt_valgrind_mysqld= 0;
 my $opt_valgrind_mysqltest= 0;
-my @default_valgrind_args= ("--show-reachable=yes");
+my @default_memcheck_valgrind_args= ("--show-reachable=yes",
```

Comment:
We don't really need it here at all, why not just add the options along with --tool=memcheck?